common/annotations/en.xml

```diff
@@ -1371,364 +1366,363 @@ annotations.
+		<annotation cp="👭">bae | bestie | bff | couple | dating | flirt | friends | girls | hand | hold | sisters | twins | women</annotation>
 		<annotation cp="👭" type="tts">women holding hands</annotation>
-		<annotation cp="👫">bae | couple | dating | everyone | flirt | friend | friends | hand | hold | holding hands | in love | man | man and woman holding hands | woman | woman and man holding hands</annotation>
+		<annotation cp="👫">bae | bestie | bff | couple | dating | flirt | friends | hand | hold | man | twins | woman</annotation>
 		<annotation cp="👫" type="tts">woman and man holding hands</annotation>
-		<annotation cp="👬">bae | boys | couple | dating | everyone | friend | friends | gay | Gemini | glbt | glbtq | hand | hold | holding hands | in love | lgbt | lgbtq | lgbtqia | man | men | men holding hands | queer | twins | two men holding hands | zodiac</annotation>
+		<annotation cp="👬">bae | bestie | bff | boys | brothers | couple | dating | flirt | friends | hand | hold | men | twins</annotation>
 		<annotation cp="👬" type="tts">men holding hands</annotation>
 		<annotation cp="💏">anniversary | babe | bae | couple | date | dating | heart | kiss | love | mwah | person | romance | together | xoxo</annotation>
 		<annotation cp="💏" type="tts">kiss</annotation>
-		<annotation cp="💑">anniversary | babe | bae | couple | couple with heart | dating | heart | kiss | love | love you | person | relationship | romance | together</annotation>
+		<annotation cp="💑">anniversary | babe | bae | couple | dating | heart | kiss | love | person | relationship | romance | together | you</annotation>
 		<annotation cp="💑" type="tts">couple with heart</annotation>
 		<annotation cp="🗣">face | head | silhouette | speak | speaking</annotation>
 		<annotation cp="🗣" type="tts">speaking head</annotation>
-		<annotation cp="👤">bust | bust in silhouette | mysterious | shadow | silhouette</annotation>
+		<annotation cp="👤">bust | mysterious | shadow | silhouette</annotation>
 		<annotation cp="👤" type="tts">bust in silhouette</annotation>
-		<annotation cp="👥">bff | bust | busts in silhouette | everyone | friend | friends | people | silhouette</annotation>
+		<annotation cp="👥">bff | bust | busts | everyone | friend | friends | people | silhouette</annotation>
 		<annotation cp="👥" type="tts">busts in silhouette</annotation>
-		<annotation cp="🫂">comfort | embrace | farewell | friendship | goodbye | hello | hug | hugging | love | people hugging | thanks</annotation>
+		<annotation cp="🫂">comfort | embrace | farewell | friendship | goodbye | hello | hug | hugging | love | people | thanks</annotation>
 		<annotation cp="🫂" type="tts">people hugging</annotation>
 		<annotation cp="👪">child | family</annotation>
 		<annotation cp="👪" type="tts">family</annotation>
-		<annotation cp="🧑‍🧑‍🧒">family: adult, adult, child</annotation> <!-- 1F9D1 200D 1F9D1 200D 1F9D2 -->
+		<annotation cp="🧑‍🧑‍🧒">adult | child | family:</annotation> <!-- 1F9D1 200D 1F9D1 200D 1F9D2 -->
```

Comment:
There is a left-over colon after "family" here.

Reply:
Ah, missed that one. I'd like to make that in a follow-on PR, because I've squashed this already. That, along with restoring Dracula, and removing Spock and Vulcan.